### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aidanraskil**
for understand the user

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # Awan
 A cloud app
+
+## Contributing
+How to contribute on this hub
```
Comment (iskandarali) [inline README.md:5]: Eleborate more on this?

Comment (aidanraskil) [reply README.md:5]: This is a new function

Comment (iskandarali) [reply README.md:5]: Ok

**Review by iskandarali — COMMENTED** [review]
can change to bahasa?

**Review by aidanraskil — COMMENTED** [review]
yes

**Review by iskandarali — COMMENTED** [review]
Ok?

**Review by aidanraskil — COMMENTED** [review]
Ok bos

**Review by iskandarali — CHANGES_REQUESTED** [review]
Pls add more line

**Review by aidanraskil — COMMENTED** [review]
Siap bos

**Review by iskandarali — APPROVED** [review]
Ok good job

**Pull request merged**